packages/appium/docs/en/sponsors.md

```diff
@@ -0,0 +1,54 @@
+---
+hide:
+  - navigation
+  - toc
+
+title: Sponsors & Backers
+---
```

Comment:
Suggest adding some CSS here to hide the sponsors footer, like on the main page:
```
<style>
  .appium-sponsor-thanks {
    display: none;
  }
</style>
```

Reply:
Updated. The description also has with this change